### ISSUE TITLE
Width is not working as expected on notifications positioned at center

### ISSUE THREAD

**Issue by BeroBurny** [teodosii/react-notifications-component#108]
property `width` provided in `addNotificitaion` is not working on property `container` properties `top-center`, `center` and `bottom-center`

![image](https://user-images.githubusercontent.com/6376047/114173284-38cad500-9937-11eb-98be-743212b37913.png)

<details>
<summary>Code used to generate notifications from screenshot</summary>

```typescript
['top-left', 'top-right', 'top-center', 'center', 'bottom-left', 'bottom-right', 'bottom-center'].forEach(container => {
      store.addNotification({
        title: "Wonderful!",
        message: "teodosii@react-notifications-component",
        type: "danger",
        insert: "top",
        container: container as ReactNotificationOptions['container'],
        dismiss: {
          duration: 10900,
        },
        width: 700,
      });
    })
```

</details>

**Comment by itsmichaeldiego** [comment]
@teodosii Same on my end! I really need this!

**Comment by itsmichaeldiego** [comment]
@teodosii I am experiencing the same issue. I did some investigation and it seems you've set a `max-width: 350px` in all the centered positions indeed:

```
/notification-container--bottom-center, .notification-container--center, .notification-container--top-center {
    display: flex;
    justify-content: center;
    align-items: center;
    flex-direction: column;
    left: calc(50% - 175px);
    max-width: 350px;
}
```

**Comment by teodosii** [comment]
Fixed in `3.1.0`, it was using `width: 100%` for centered containers instead of the supplied width.
